.github/workflows/upstream_testing.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: "Nautobot Upstream Testing"
+
+on: # yamllint disable-line rule:truthy rule:comments
+  schedule:
+    - cron: "0 4 */2 * *"  # every other day at midnight
+  push:  # yamllint disable rule:empty-values
+# TODO: REMOVE PUSH LINE ABOVE
+
+jobs:
+  upstream-test:
+    uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@2023-app-ecosystem-ci-testing-against-core"  # TODO: Update to core base once available
+    with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
+      invoke_context_name: "NAUTOBOT_GOLDEN_CONFIG"
+      plugin_name: "nautobot-plugin-golden-config"
```

Comment:
Looks like maybe this needs to be `nautobot-golden-config` instead? The `build` stage and the `unittest` stage are using different Docker tags because of this.

Reply:
Adopted fix from #354